catalogbuilder/intakebuilder/dat/gfdlcmipfreq.yaml

```diff
@@ -1,5 +1,7 @@
 monthly:
     frequency: mon
+h0:
+    frequency: mon
```

Comment:
CESM files need additional translations for different history file strings (h1,h2,...) and the frequency associated with them. Unfortunately the association is made by the person who builds the case, so it might have be to be changed until we start opening files and looking at the file attribute 
> time_period_freq  = "day_1" ;
For the example files, h2 is daily so we can add this:
h2:
    frequency: day